content/en/ddsql_reference/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: DDSQL Reference
+type: multi-code-lang
+external_redirect: /ddsql_reference/ddsql_default
+further_reading:
+- link: "/ddsql_editor"
+  tag: "Documentation"
+  text: "Learn more about the DDSQL Editor"
+---
+
+## Supported DDSQL Syntax
+
+DDSQL is a query language for Datadog data. It implements several standard SQL operations, such as `SELECT`, and allows queries against unstructured data, such as [tags][1]. Get exactly the data you want by writing your own `SELECT` statement. Query tags as if they are standard table columns.
```

Comment:
```suggestion
DDSQL is a query language for Datadog data. It implements several standard SQL operations, such as `SELECT`, and allows queries against unstructured data, such as [tags][1]. You can perform actions like getting exactly the data you want by writing your own `SELECT` statement, or querying tags as if they are standard table columns.
```
Really minor suggestions here to make this flow a little better - totally optional 🙂